.github/workflows/leetcode_directory_writer.yml

```
@@ -0,0 +1,24 @@
+# The objective of this GitHub Action is to update the leetcode DIRECTORY.md file (if needed)
+# when doing a git push
+name: leetcode_directory_writer
+on: [push]

```

We should run this only on the LeetCode directory.
```suggestion
on:
  push:
    paths:
      - "leetcode/src/**.c"
```